ntbs-service-unit-tests/Helpers/EpisodesExtensionMethodsTest.cs

```diff
@@ -59,17 +63,22 @@ private void AssertTreatmentEventOrder(List<TreatmentEvent> orderedList)
             Assert.Equal(TreatmentEventType.TreatmentStart, orderedList[1].TreatmentEventType);
             Assert.Equal(TreatmentEventType.TransferOut, orderedList[2].TreatmentEventType);
             Assert.Equal(TreatmentEventType.TransferIn, orderedList[3].TreatmentEventType);
-            Assert.Equal(TreatmentEventType.TreatmentRestart, orderedList[4].TreatmentEventType);
-            Assert.Equal(TreatmentEventType.TreatmentOutcome, orderedList[5].TreatmentEventType);
-            Assert.All(orderedList.GetRange(0, 6), ev => Assert.Equal(ev.EventDate, earlierDate));
+            Assert.Equal(TreatmentEventType.TransferOut, orderedList[4].TreatmentEventType);
+            Assert.Equal(TreatmentEventType.TransferIn, orderedList[5].TreatmentEventType);
+            Assert.Equal(TreatmentEventType.TreatmentRestart, orderedList[6].TreatmentEventType);
+            Assert.Equal(TreatmentEventType.TreatmentOutcome, orderedList[7].TreatmentEventType);
+            Assert.All(orderedList.GetRange(0, 2), ev => Assert.Equal(ev.EventDate, earlierDate));
+            Assert.All(orderedList.GetRange(2, 2), ev => Assert.Equal(ev.EventDate, earlierDateEarlierTime));
+            Assert.All(orderedList.GetRange(4, 2), ev => Assert.Equal(ev.EventDate, earlierDateLaterTime));
+            Assert.All(orderedList.GetRange(6, 2), ev => Assert.Equal(ev.EventDate, earlierDate));
```

Comment:
This test has become rather bloated (i.e. it's testing all of the ordering logic in one go, which could make failures more difficult to debug, and also makes maintaining the test harder). It might be worth splitting it up into separate tests. But I'm also happy for you to leave that as tech debt.